.github/renovate.json

```
@@ -11,6 +11,10 @@
             "groupName": "github-actions",
             "matchManagers": ["github-actions"],
             "separateMajorMinor": "false"
+        },
+        {
+            "groupName": "requirements-cibw.txt",
+            "matchPaths": [".ci/requirements-cibw.txt"]
```

```suggestion
            "matchFileNames": [".ci/requirements-cibw.txt"]
```
https://docs.renovatebot.com/release-notes-for-major-versions/
> matchPaths and matchFiles are now combined into matchFileNames, supporting exact match and glob-only.